packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1674,6 +1674,8 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
     _cursorVisibilityNotifier.value = widget.showCursor;
   }
 
+  bool _tickersEnabled = true;
```

Comment:
Nit: Maybe add a comment explaining this like you did for _cursorActive, if you think it would be helpful.

Reply:
Added.